src/templates/core/proposals/assign/start_proposal.html

```diff
@@ -39,25 +39,30 @@
         {% bootstrap_form default_fields %}
         {% if proposal_form %}
           <div class="row">
-         {% for field in proposal_form %}
-            <div class="{% if field.field.widget.attrs.div_class %}{{ field.field.widget.attrs.div_class }}{% else %}col-md-12{% endif %}">
+          {% for field in proposal_form %}
+            <div class="{% if field.field.widget.attrs.div_class %}{{ field.field.widget.attrs.div_class }}
+            {% else %}col-md-12{% endif %}">
             {% if field.field.widget.attrs.is_checkbox %}
-              {% if field.errors %}<span class="red">{{ field.errors|join:", " }}</span><br />{% endif %}
-              {{ field }} &nbsp;{% if field.field.help_text %}{{ field.field.help_text|safe }}{% endif %}
-            {% else %}
-                 {% if field.errors %}<span class="red">{{ field.errors|join:", " }}</span>{% endif %}
-                 {% bootstrap_field field %}
+              {% if field.errors %}
+                <span class="red">{{ field.errors|join:", " }}</span><br />
+              {% endif %}
+              {{ field }} &nbsp;
+              {% if field.field.help_text %}
+                {{ field.field.help_text|safe }}
               {% endif %}
-              </div>
+            {% else %}
+                {% bootstrap_field field %}
+            {% endif %}
+            </div>
           {% endfor %}
           </div>
-          {% endif %}
-        </div>
+        {% endif %}
+      </div>
 
         
 	    {% buttons %}
       <div class="dash-container">
-	        <button name="book" type="submit" class="btn btn-other pull-right" style="width:120px">
+	        <button name="book" type="submit" formnovalidate class="btn btn-other pull-right" style="width:120px">
```

Comment:
The way Chrome interacts with HTML5 means that it automatically adds 'Please fill out this field' tags to required fields that messed with our existing validation. These tags disable that.